frontend/build.rs

```diff
@@ -0,0 +1,107 @@
+///
+/// Build Script
+/// This is run as a pre-build step -- before the rust backend is compiled.
+/// NOTE: Should be included in root's build script
+///
+use std::ffi::OsStr;
+use std::fmt::Display;
+use std::fs::File;
+use std::path::{Path, PathBuf};
+use std::{fs, io};
+use std::{io::Write, process::Command};
+
+#[cfg(target_family = "windows")]
+const SHELL: &str = "cmd";
+#[cfg(target_family = "unix")]
+const SHELL: &str = "sh";
+#[cfg(target_family = "windows")]
+const SHELL_ARG: &str = "/C";
+#[cfg(target_family = "unix")]
+const SHELL_ARG: &str = "-c";
+
+/*
+ * Note 1: this file was written for *nix systems -- it likely won't
+ * work on windows!
+ */
+
+#[allow(dead_code)]
+fn shell(command: impl AsRef<OsStr> + Display) {
+    // println!("build.rs => {}", command);
+
+    let output = Command::new(SHELL)
```

Comment:
I think shell is not needed. You can run directly the app, that passed as a command (at least `yarn` can be run directly)

Reply:
on Windows that doesn't work, for some reason, at least for me. For me it throws `Unknown command` on `yarn` call. Maybe that's a me problem, not a shell problem, not really sure

Reply:
I've checked the problem: that's because on Windows `yarn` is a `cmd` script and can be run only with `cmd`. So, let's keep the logic you have already implemented